auditlog/models/rule.py

```diff
@@ -294,9 +284,13 @@ def create_logs(self, uid, res_model, res_ids, method,
         log_model = self.env['auditlog.log']
         for res_id in res_ids:
             model_model = self.env[res_model]
-            res_name = model_model.browse(res_id).name_get()
+            # Avoid recursivity with the 'read' method called by 'name_get()'
+            res_name = "%s,%s" % (res_model, res_id)
+            if method is not 'read':
+                name = model_model.browse(res_id).name_get()
+                res_name = name and name[0] and name[0][1] or res_name
```

Comment:
a possibility would be to put some key into the context that causes read above not to do any logging. And given you use the new api here, you'd only have to check that in the v8 branch